### DISCUSSION TITLE
Compilation error for tool load v1.5.x

### DISCUSSION THREAD

**Discussion by alexmkrolak** [visose/Robots#173] [Q&A] [answered]
I was previously on v1.3.x and updated to v1.5.x and there this difference in the .dat files generated by Robots. The extruder file on v1.5.x has the DECL GLOBAL LOAD line for the tool which seems to cause issues when compiling on the KRC4 with KSS 8.3.17. Just wondering if its intentional and how to avoid getting the error.

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/69169386/182379435-a79c32a3-eefb-4b77-a167-3942953a829d.png)

**Comment by visose** [comment, marked as answer]
Hi @alexmkrolak, this was fixed in the last release from just a few days ago. Check in the package manager if you have a new update available.

**Reply by alexmkrolak** [reply]
Right, yes, thank you!

**Comment by visose** [comment]
Btw, there's another less harmful bug where if you connect a panel to a list of targets, the text will say they contain commands when they don't. This is fixed but not published yet.
